Annotato/Annotato/Views/Document/Edit/DocumentPdfView.swift

```diff
@@ -16,3 +16,15 @@ class DocumentPdfView: PDFView {
         self.document = viewModel.document
     }
 }
+
+extension DocumentPdfView {
+    func visiblePagesContains(view: UIView) -> Bool {
+        guard let centerInDocument = self.documentView?.convert(view.center, to: self) else {
+            return false
+        }
+        guard let pageContainingView = self.page(for: centerInDocument, nearest: true) else {
+            return false
+        }
+        return self.visiblePages.contains(pageContainingView)
+    }
```

Comment:
is it possible for this to just be an extension on `PDFView` instead of our `DocumentPdfView`? Can put in the views > extensions folder

Reply:
Ok I will update